pom.xml

```diff
@@ -174,6 +175,13 @@
             <version>${version.org.wildfly.common}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.kohsuke.metainf-services</groupId>
+            <artifactId>metainf-services</artifactId>
+            <version>${version.org.kohsuke.metainf-services.metainf-services}</version>
+            <scope>provided</scope>
+        </dependency>
+
```

Comment:
It looks like this dependency is unused?

Reply:
Thanks for catching that! 
I have removed the dependency.